### PR TITLE
Update and rename Visa.xml to Visa.com.xml

### PR DESCRIPTION
**Pull request by ivysrono**

### CODE REVIEW

File: src/chrome/content/rules/Visa.com.xml [hunk 1]
```diff
@@ -0,0 +1,55 @@
+<!--
+	Other Visa rulesets:
+		- Visa_Buxx.com.xml
+		- Visa_Europe.xml
+
+	403:
+		developer.visa.com
```
Comment (J0WI) [inline src/chrome/content/rules/Visa.com.xml]: Works for me

File: src/chrome/content/rules/Visa.com.xml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+		(www.)?visa\.com/atmlocator/
+		benefitsource.visa.com
+		cis.visa.com
+
+	Invalid certificate:
+		bd.visa.com
+		design.visa.com
+		drjc1.visa.com
+		rvcom-qa.visa.com
+		rvcom-qa.vc.visa.com
```
Comment (J0WI) [inline src/chrome/content/rules/Visa.com.xml]: investor.visa.com

File: src/chrome/content/rules/Visa.com.xml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+	<target host="community.developer.visa.com" />
+	<target host="icicibank.intellilink.visa.com" />
+	<target host="loyaltyoffer.visa.com" />
+	<target host="multinationalpremiumsolutions.visa.com" />
+	<target host="promociones.visa.com" />
+	<target host="research.visa.com" />
+	<target host="smail.visa.com" />
+	<target host="technologypartner.visa.com" />
+	<target host="travel.visa.com" />
+	<target host="usa.visa.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/Visa.com.xml]: aw.visa.com
purchasealerts.visa.com

File: src/chrome/content/rules/Visa.com.xml (hunk 1 above)
Comment (ivysrono) [inline src/chrome/content/rules/Visa.com.xml]: Me too.
However, https://travis-ci.org/EFForg/https-everywhere/jobs/207362756#L218

File: src/chrome/content/rules/Visa.com.xml [hunk 4]
```diff
@@ -0,0 +1,55 @@
+<!--
+	Other Visa rulesets:
+		- Visa_Buxx.com.xml
+		- Visa_Europe.xml
```
Comment (J0WI) [inline src/chrome/content/rules/Visa.com.xml]: This will change to VisaEurope.xml in #8881

File: src/chrome/content/rules/Visa.com.xml (hunk 1 above)
Comment (J0WI) [inline src/chrome/content/rules/Visa.com.xml]: Maybe you can use a static resource as test url, like `https://developer.visa.com/assets/css/portal.css`

Comment (ivysrono) [inline src/chrome/content/rules/Visa.com.xml]: 403：https://travis-ci.org/EFForg/https-everywhere/jobs/207389772#L221

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**